docs/plugins/AliasRedirects.md

```diff
@@ -0,0 +1,18 @@
+---
+title: AliasRedirects
+tags:
+  - plugin/emitter
+---
+
+This plugin generates HTML redirect pages for aliases and permalinks defined in the frontmatter of content files, so that alternative paths correctly lead to the intended content.
+
+> [!note]
+> For information on how to add, remove or configure plugins, see the [[Configuration#Plugins|Configuration]] page.
+
+This plugin has no configuration options.
+
+## Technical Details
+
+- Category: Emitter
+- Function name: `Plugin.AliasRedirects()`.
+- Source: [`quartz/plugins/emitters/aliases.ts`](https://github.com/jackyzha0/quartz/blob/v4/quartz/plugins/emitters/aliases.ts).
```

Comment:
Sure, I can change that if you prefer "API".

I put in the category here because it's easier to spot next to the function name and the user needs to put the function into the right place in the config file. wdyt?

Reply:
Oh I mean the source is still helpful, not sure about the `category` and `function name`, since the page itself already is the function name, and also `Plugin.AliasRedirects` here is mainly a typescript import.

So I don't think it is helpful at all.

Reply:
Changed to API but left the caterogy for now until confirmation.